app/controllers/notebooks_controller.rb

```diff
@@ -136,12 +139,25 @@ def update
     @old_content = @notebook.content
     @tags = parse_tags
     populate_notebook
-
-    # Save the content and db record.
-    if save_update
+    errors = " "
+    summary = params[:summary].strip
+    if summary.length > 500
+      errors += "Change log was too long. Only accepts 500 characters and you tricked the form to submit one that was #{summary.length} characters."
```

Comment:
Don't say "you tricked the form".  Just leave it as "500 characters and you submitted on that was ...."

Reply:
Gotcha gotcha. Makes sense. More professional

Reply:
Not just that, it automatically assigns blame to the user when at some point a Javascript glitch could occur that allows it through anyway.

Reply:
I just assume people that get crazy bugs are just doing crazy things, probably a bunch of hackers. Makes sense though. Will make the changes